src/rs.cpp

```diff
@@ -388,6 +389,79 @@ void rs2_get_video_stream_intrinsics(const rs2_stream_profile* from, rs2_intrins
 }
 HANDLE_EXCEPTIONS_AND_RETURN( , from, intr )
 
+void rs2_aus_set( const char * counter, int value, rs2_error ** error ) BEGIN_API_CALL
+{
+    VALIDATE_STRING( counter );
+    librealsense::aus_set( counter, value );
+}
+NOARGS_HANDLE_EXCEPTIONS_AND_RETURN_VOID()
+
+void rs2_aus_increment( const char * counter, rs2_error ** error ) BEGIN_API_CALL
+{
+    VALIDATE_STRING( counter );
+    librealsense::aus_increment( counter );
+}
+NOARGS_HANDLE_EXCEPTIONS_AND_RETURN_VOID()
+
+void rs2_aus_decrement( const char * counter, rs2_error ** error ) BEGIN_API_CALL
+{
+    VALIDATE_STRING( counter );
+    librealsense::aus_decrement( counter );
+}
+NOARGS_HANDLE_EXCEPTIONS_AND_RETURN_VOID()
+
+long rs2_aus_get( const char * counter, rs2_error ** error ) BEGIN_API_CALL
+{
+    VALIDATE_STRING( counter );
+    return librealsense::aus_get( counter );
+}
+NOARGS_HANDLE_EXCEPTIONS_AND_RETURN( 0 );
+
+void rs2_aus_start( const char * timer, rs2_error ** error ) BEGIN_API_CALL
+{
+    VALIDATE_STRING( timer );
+    librealsense::aus_start( timer );
+}
+NOARGS_HANDLE_EXCEPTIONS_AND_RETURN_VOID()
+
+void rs2_aus_stop( const char * timer, rs2_error ** error ) BEGIN_API_CALL
+{
+    VALIDATE_STRING( timer );
+    librealsense::aus_stop( timer );
+}
+NOARGS_HANDLE_EXCEPTIONS_AND_RETURN_VOID()
+
+
+const rs2_strings_list * rs2_aus_get_counters_list( rs2_error ** error ) BEGIN_API_CALL
```

Comment:
We should think about returning a json file here,
Than we can return a single string with all of the data, thoughts?

Reply:
added to the TODO list